### PR TITLE
Fed 842

### PR DESCRIPTION
**Pull request by meuschke**
- own middleware for grunt dev server
- remove "winston-posix-syslog" because it do not work with node > v4 (waiting for worker remove)
- add some polyfills

### CODE REVIEW

**Comment by zcei** [comment]
What's the plan for removing worker?
We have an updated version (node v4.2) in GitLab

Also there is a PR open (#41) for removing `winston` completely, which will be substituted by `bunyan`

**Comment by meuschke** [comment]
I think if there are no dependencies we should remove that asap! Lets discuss that tomorrow.

**Comment by meuschke** [comment]
The winston logger seems to be removed from master. So could we merge that pull-request?

File: Gruntfile.js [hunk 1]
```diff
@@ -309,10 +310,10 @@ module.exports = function (grunt) {
             server: {
                 options: {
                     keepalive: true,
-                    open: true,
+                    //open: true,
```
Comment (zcei) [inline Gruntfile.js]: Can we remove this line?

File: Gruntfile.js [hunk 2]
```diff
@@ -398,6 +399,21 @@ module.exports = function (grunt) {
                                 next();
                             }
                         });
+
+                        // get the own middleware module
+                        if(ownMiddleware){
+                           try {
+                             ownMiddlewareFn = require(ownMiddleware);
+                             if(typeof ownMiddlewareFn !== "function"){
+                               throw new Error("");
```
Comment (zcei) [inline Gruntfile.js]: `throw new Error("Middleware must export function: " + require.resolve(ownMiddleware));`

File: lib/view-helpers/view-helper-object.js [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
-var viewHelperObject = {
+
+// Hack to cheat on destilify and still have the posibility to export the object
+if (typeof steal === 'undefined') {
+  steal = function (cb) {
+    var externalViewHelperObject = cb.call(this)
+    if (typeof module !== 'undefined' && typeof module.require === 'function') {
+      var _ = require('lodash')
+      module.exports = function (res) {
+        // config = res
```
Comment (zcei) [inline lib/view-helpers/view-helper-object.js]: Can we remove this line?

**Pull request merged**